src/test/java/org/commcare/formplayer/tests/BaseTestClass.java

```diff
@@ -1035,7 +1041,8 @@ protected FormSession getFormSession(SerializableFormSession serializableFormSes
                 storageFactoryMock,
                 getCommCareSession(serializableFormSession.getMenuSessionId()),
                 remoteInstanceFetcher,
-                formDefinitionService
+                formDefinitionService,
+                "1796"
```

Comment:
It's better to have the individual test pass it instead of harcoding it in a parent test class.